apps/steercode-app-electron/src/renderer/src/lib/features/ConversationThread/layout/ChatThreadInner.svelte

```diff
@@ -22,12 +23,16 @@
     initialFileTreeFile,
     selectedRepositoryStore
   } from '$lib/shared/stores/selectedRepository';
+  import { openModal } from '$lib/features/SubscribeModal/layout/SubscribeModal.svelte';
+  // import { openModal } from '$lib/features/SettingsModal/layout/SettingsModal.svelte';
 
   export let conversation: Conversation;
   let loading: boolean = false;
   let answer: string = '';
   let wrapContainer: ConversationWrapper;
   let eventSource: SSE | null = null;
+  const streamController = new AbortController();
```

Comment:
Initialize AvortController in the `handleSubmit` function otherwise once you abort the AC all subsequent "handleSubmit" calls will get a signal from the AC that is already aborted.

`let streamController: AbortController;`

then in the "handleSubmit" function
```javascript
async function handleSubmit(query: string) {
  ...
  streamController = new AbortController()
  const signal = streamController.signal;
  ...
```